docs/references/expo/local-credentials.mdx

```diff
@@ -0,0 +1,235 @@
+---
+description: Learn how to use local authentication in your Expo app with Clerk.
+---
+
+# Use biometrics with local credentials
+
+This guide shows you how to take advantage of local authentication and allow users that have **already signed in** once to re-sign in by using the biometric authenticator from their device.
+
+> [!WARNING]
+> This feature requires [@clerk/clerk-expo v2](/docs/upgrade-guides/expo-v2/upgrade).
+> Be aware that this works only for sign in attempts with the password strategy
+
+<Steps>
+  ### Install the necessary peer depedencies
+
+  These packages are required to be installed in order to use the `useLocalCredentials` hook.
+
+  <CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+    ```bash {{ prettier: false, filename: 'terminal' }}
+    npm install expo-local-authentication expo-secure-store
+    ```
+
+    ```bash {{ prettier: false, filename: 'terminal' }}
+    yarn add expo-local-authentication expo-secure-store
+    ```
+
+    ```bash {{ prettier: false, filename: 'terminal' }}
+    pnpm add expo-local-authentication expo-secure-store
+    ```
+  </CodeBlockTabs>
+
+  ### Update `app.json`
+
+  Update your app.json file as instructed in the expo documentation
+
+  - [expo-local-authentication](https://docs.expo.dev/versions/latest/sdk/local-authentication/#configuration-in-appjsonappconfigjs)
+  - [expo-secure-store](https://docs.expo.dev/versions/latest/sdk/securestore/#configuration-in-appjsonappconfigjs)
+
+  ### Securely store/access the user's credentials during sign in
+
+  The following example demonstates how to use `useLocalCredentials()` in a custom flow for signing users in.
+
+  ```tsx {{ prettier: false, filename: '/app/sign-in.tsx' }}
```

Comment:
Please can we remove `prettier: false` from all of the code blocks, including the `bash` ones? If there's specific formatting that needs to maintained you can use [prettier-ignore comments](https://prettier.io/docs/en/ignore.html)